tutorials/500_spatiotemporal.ipynb

```diff
@@ -7,23 +7,43 @@
    "source": [
```

Comment:
<p>Do we want to keep all these variables, e.g. why do we have obs columns of the form <code>{celltype}_annotation</code>,also not sure we need all of the <code>_colors</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>its not necessary to save all - it was for the per-organ mapping. guess the object can be modified.</p>

Reply:
I would remove the unnecessary data from the anndata object.

Reply:
sharing the updated object for you to upload